web/src/components/Editor/Expression/Expression.tsx

```diff
@@ -0,0 +1,63 @@
+import {EditorView, hoverTooltip} from '@codemirror/view';
+import {autocompletion} from '@codemirror/autocomplete';
+import CodeMirror from '@uiw/react-codemirror';
```

Comment:
Each editor has its own folder with the specific syntax, hooks and functionality